lib/core.js

```diff
@@ -300,6 +300,15 @@ interface Iterable<T> {
     @@iterator(): Iterator<T>;
 }
 
+/* Generators */
+interface Generator<Y,R,N> {
+    @@iterator(): Iterator<Y>;
+    next(value?: N): IteratorResult<R|Y>;
```

Comment:
!!!

That did it! I also get a much nicer error message for one of the unexpected failures.